### ISSUE TITLE
[1.18.2] Crash on world load

### ISSUE THREAD

**Issue by xAlicatt** [pale-imitations/schools_of_magic#25]
Crash log: https://paste.ee/p/dZmkW

**Comment by xAlicatt** [comment]
This crash seems to be happening with Forge 40.2.1, but not on 40.2.0

**Comment by festro** [comment]
Getting the same thing, so rather than creating another thread hope you don't mind me posting here too
Here's my crash report
[crash-2023-03-12_16.45.13-client.txt](https://github.com/pale-imitations/schools_of_magic/files/10952427/crash-2023-03-12_16.45.13-client.txt)

**Comment by CFlatFlat** [comment]
I get the same issue, meaning I can't use the mod together with Valkyrian Skies 2 and the new releases of Fossils and Archeology. My guess is that the mod relied on a deprecated function/feature that got removed in 40.2.1

**Comment by xAlicatt** [comment]
> I get the same issue, meaning I can't use the mod together with Valkyrian Skies 2 and the new releases of Fossils and Archeology. My guess is that the mod relied on a deprecated function/feature that got removed in 40.2.1

Yeah and it seems like this mod is currently abandoned. There are better mods at this point, though nothing with quite the same theming.

**Comment by CFlatFlat** [comment]
I tried forking the mod to look into the issue, but for the life of me I can't get the necessary Minecraft java files loaded as an external library for the project, which would be necessary. 

There are certainly better magic mods, but this one is pretty unique, and is precisely what I'd want to use with Fossils and Archeology on this version (which only just a month ago got finished porting past 1.12.2, onto 1.18). Mana and Artifice is awesome, but would mesh poorly with Fossils for a lot of reasons. I think Iron's Spells and Spellbooks lacks too much of its content on this version to be worth using. Ars Nouveau is bland and uninteresting, and its mobs are a nuisance more than anything. 

Where's Thaumcraft when you need it?

**Comment by CFlatFlat** [comment]
Well no wonder! The source code is only available for the 1.16.5 version, not the 1.18.2 version.
